lib/clockwork/test/manager.rb

```diff
@@ -46,6 +41,19 @@ def block_for(job)
 
       private
 
+      def set_opts(opts)
+        @max_ticks = opts[:max_ticks] if opts[:max_ticks]
+        @start_time = opts[:start_time] if opts[:start_time]
+        if opts[:end_time]
+          @end_time = opts[:end_time]
+        elsif @start_time
+          @end_time = Time.current
+        end
+        @tick_speed = opts[:tick_speed]
+
+        raise "End time can't precede start time." if @start_time && (@start_time > @end_time)
```

Comment:
Minor point, but on this line, and also in `elsif @start_time`, if we have the `attr_reader`s for the i-vars available, I'd rather reference the value through the method than calling to the i-var directly.

Reply:
I don't understand what you mean by saying the same variable is used two different ways on this line - but I'm all for making sure this is not confusing.

For clarity, I'm thinking: 
```
raise "End time can't precede start time." if start_time && (start_time > end_time)
```

Reply:
Three lines up (46-50) we reference this values as `@start_time`, because we're setting it there.

This looks weird for me:
```
@start_time = opts[:start_time]
...
raise "End time can't precede start time." if start_time
```

And it's not obvious that those are the same variables.